pages/docs/how-to-guides/events.mdx

```diff
@@ -41,7 +41,10 @@ _If you need help on how to edit this file, please see the <Link href="/docs/how
     "city": "Jafféstrasse",
     "state": "Berlin",
     "country": "Germany"
-  }
+  },
+  // optional
```

Comment:
I am not sure about putting a comment in here.  Comments are not valid in JSON so if someone grabs this unknowingly they would be copying something invalid. wdyt?

Reply:
Ah, I did not realize that. Will update it.

Reply:
Looks great, thanks!